Header header

#yaw_speed [int] -100~0~100. away from zero -> fast, close to zero -> slow. Sign is for direction
#pitch_speed [int] Same as yaw_speed

int8 yaw_speed    
int8 pitch_speed